src/shared/components/dateRangePicker/DatePicker.tsx

```diff
@@ -77,7 +77,7 @@ class DatePicker extends PureComponent<Props, State> {
   private inCurrentMonth: boolean = false
   state = {
     inputValue: null,
-    inputFormat: null,
+    inputFormat: 'YYYY-MM-DD HH:mm:ss',
```

Comment:
tip for the future: before using a string like a dateformat, or instuctrions, do a global search for it. I found [this](https://github.com/influxdata/ui/blob/master/src/utils/datetime/constants.ts#L1), which should probably be used here 👍